utility/module.go

```diff
@@ -143,16 +141,32 @@ func (u *utilityModule) GetActorModules() map[string]modules.Module {
 	return u.actorModules
 }
 
-func (u *utilityModule) GetServicerModule() modules.ServicerModule {
-	return u.actorModules[servicer.ServicerModuleName]
+func (u *utilityModule) GetServicerModule() (modules.ServicerModule, error) {
+	m, ok := u.actorModules[servicer.ServicerModuleName].(modules.ServicerModule)
+	if !ok {
+		u.logger.Warn().Err(errors.New("failed to cast servicer module")).Msg("failed to get servicer module")
+		return nil, errors.New("failed to cast servicer module")
+	}
+	return m, nil
 }
 
-func (u *utilityModule) GetFishermanModule() modules.FishermanModule {
-	return u.actorModules[fisherman.FishermanModuleName]
+func (u *utilityModule) GetFishermanModule() (modules.FishermanModule, error) {
+	m, ok := u.actorModules[fisherman.FishermanModuleName].(modules.FishermanModule)
```

Comment:
Can we ever be in a situation where the map does not contain the key so we get a panic because we try to typeast a nill?

Reply:
Currently not, but you're right, it is sort of shoot urself in the foot material. Added more checks.